packages/sitecore-jss/src/personalization/layout-personalization-utils.ts

```diff
@@ -0,0 +1,123 @@
+import {
+  PersonalizedComponentRendering,
+  isComponentRendering,
+  isPersonalizedComponentRendering,
+  ComponentRendering,
+  PlaceholdersData,
+  hasPersonalization,
+  HtmlElementRendering,
+} from '../layout/models';
+
+export class LayoutPersonalizationUtils {
```

Comment:
Does it make sense to export list of functions instead of a class (as they are stateless)?

Reply:
We have discussed this possibility and decided use separate class approach